experimental/pages/demologin/views/demologin_view.dart

```diff
@@ -48,17 +48,20 @@ class DemologinView extends GetView<DemologinController> {
                     onPressed: () async {
                       // バリデーションをパスできた場合
                       if(_formKey.currentState?.saveAndValidate() ?? false) {
-                        //TODO: 認証の判定
-                        if(await Future.value(true)){
-                          Get.offNamed(DemoRoutes.demoHome);
+                        DemoUser? user = await controller.authentication(_formKey.currentState?.value[_fieldNameEmail] ?? "");
+                        // ユーザーを取得できない場合
+                        if(user == null){
+                          _formKey.currentState?.invalidateField(name: _fieldNameEmail, errorText: "メールアドレスが存在しません。");
+                          return ;
                         }
+
+                        Get.offNamed(DemoRoutes.demoHome, arguments: user); // ホーム画面へ遷移
```

Comment:
```suggestion
                        Get.offNamed(DemoRoutes.demoHome, arguments: user); // ホーム画面へ遷移
                        return;
```
`return`入れておいてほしいです。

Reply:
かしこまりました！
入れる意味としては、この上にブロック節をパスした場合は
処理が`Get.offNamed(DemoRoutes.demoHome, arguments: user);`のみということを強調するためでしょうか？

この文の下のelse節にも入れたほうがいいのかわからなかったので、ご確認させていただきたいです。
(関数には必ず`return`をつけるルールとかでしょうか？)

Reply:
@RikitoNoto 
ご確認ありがとうございます！確かにそうですね！else節にも入れた方がいい気もしてきますね。returnを入れた方がここで処理が終わってると明示的に示せるので、私の好みですがどっちもreturnを入れちゃいますね。なので、Notoさんが今後もしここの処理を変更するときに迷わなければreturn入れなくてもいいですよ。